drivers/scheduler/k8s/k8s.go

```diff
@@ -717,8 +717,20 @@ func (k *K8s) createStorageObject(spec interface{}, ns *v1.Namespace, app *spec.
 		}
 		if autopilotEnabled {
 			apRule := options.AutopilotRule
+			labels := options.Labels
 			if apRule.Name != "" {
-				aRule, err := k.createAutopilotRule(apRule, options)
+				apRule.Labels = defaultTorpedoLabel
+				apRule.Spec.Selector = apapi.RuleObjectSelector{
+					LabelSelector: metav1.LabelSelector{
+						MatchLabels: labels,
+					},
```

Comment:
since we are using the same selector here and in NamespaceSelector, save in a variable and use that. This will help avoid divergence.